rmw_connext_cpp/src/functions.cpp

```diff
@@ -63,7 +63,6 @@ _create_type_name(
 
 extern "C"
 {
-
```

Comment:
Seems a bit strict to me. I don't mind minimizing whitespace, i.e. warning when more than one line is between items, but this kind of sucks.